test/topology_custom/test_lwt_semaphore.py

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import asyncio
+import time
+from test.pylib.rest_client import inject_error
+from test.pylib.util import wait_for_cql_and_get_hosts
+import pytest
+from cassandra.protocol import WriteTimeout
+
+@pytest.mark.asyncio
+async def test_cas_semaphore(manager):
```

Comment:
Please add a docstring saying that this is a regression test for `scylladb/scylladb#19698`.

Reply:
Does it fail before the patch?

Reply:
It does.